__tests__/app.test.js

```diff
@@ -212,3 +212,103 @@ describe("6. GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("7. POST /api/articles/:article_id/comments/", () => {
+  it("status:200, should responds with the new comment objects if user exists", () => {
+    const newPost = {
+      username: "butter_bridge",
+      body: "test body",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newPost)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment).toEqual(
+          expect.objectContaining({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: 1,
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          })
+        );
+      });
+  });
+
+  it("status:404, should responds with error message when user does not exists in database but still post comment", () => {
```

Comment:
Can I check what you mean by "still post comment"?

Reply:
My mistake, I should add in 'still trying to post comment' instead